### PR TITLE
[msal-node] Fix silentLogin and logout in Electron sample

### LINKED ISSUE #2729: Delete cache on logOut msal-node electronTestApp
I'm working on your sample  [ElectronTestApp](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-node-samples/standalone-samples/ElectronTestApp and the log out doesn't work poperly, when you log out it actually doesn't delete the cache and I can still do a silent login after logout, instead I should be redirect to the login page
).

It seems like  `removeAccount` doesn't delete the account. is there anyway to delete the cache and be redirected to the login page?
```
this.clientApplication
          .getTokenCache()
           .removeAccount(this.account);
```

## Library
- [x] `@azure/msal-node@1.0.0-beta.2`

## Framework
- [x] electron

### PR DESCRIPTION
**Pull request by hectormmg**
This PR:

- Fixes a bug in the Electron Sample App that prevented the cached account being cleared during logout
- Adds usage README
- Fixes #2729

### CODE REVIEW

File: samples/msal-node-samples/standalone-samples/ElectronTestApp/src/AuthProvider.ts [hunk 1]
```diff
@@ -135,12 +135,16 @@ export default class AuthProvider {
...
+            this.account = await this.getAccount();
+        }
+
+        return this.account;
     }
 
     async logout(): Promise<void> {
         if (this.account) {
-            await this.clientApplication.getTokenCache().removeAccount(this.account);
+            this.clientApplication.getTokenCache().removeAccount(this.account);
```
Comment (tnorling) [inline samples/msal-node-samples/standalone-samples/ElectronTestApp/src/AuthProvider.ts:147]: Why is this no longer awaited?

**Comment by jasonnutter** [comment]
I dont see the README, did you forget to commit?

File: samples/msal-node-samples/standalone-samples/ElectronTestApp/src/AuthProvider.ts (hunk 1 above)
Comment (hectormmg) [reply samples/msal-node-samples/standalone-samples/ElectronTestApp/src/AuthProvider.ts:147]: Thanks for catching it, I undid changes and forgot to put it back. I'll put it back.

**Comment by hectormmg** [comment]
> I dont see the README, did you forget to commit?

Sorry, wrote the description before writing the README when I pushed the bugfix.

**Review by tnorling — APPROVED** [review]

**Review by jasonnutter — APPROVED** [review]

File: samples/msal-node-samples/standalone-samples/ElectronTestApp/README.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+```bash
+$ git clone https://github.com/AzureAD/microsoft-authentication-library-for-js.git
+```
+
+You can also download the repository as a zip file by selecting "Download ZIP" from the root repository's dropdown "Code" menu. Once you've downloaded the ZIP file, you can decompress it locally and explore the code.
+
+### Pre-requisites
+- By using MSAL Node, you are working with the Microsoft Identity ecosystem. Read about [App Registrations](https://docs.microsoft.com/en-us/graph/auth-register-app-v2) and register one for use with this code.
+- Install [Node.js](https://nodejs.org/en/), [Electron.js](https://www.electronjs.org/) and[TypeScript](https://www.typescriptlang.org/) if needed.
```
Comment (sameerag) [inline samples/msal-node-samples/standalone-samples/ElectronTestApp/README.md:31]:
```suggestion
- Install [Node.js](https://nodejs.org/en/), [Electron.js](https://www.electronjs.org/) and [TypeScript](https://www.typescriptlang.org/) if needed.
```

**Review by sameerag — APPROVED** [review]

**Comment by Jack-Barry** [comment]
I've tried running this app with the fix included but it's still not prompting for credentials if I log out then click log in, is there anything else I need to be on the lookout for to make sure that doesn't happen?

**Pull request merged**